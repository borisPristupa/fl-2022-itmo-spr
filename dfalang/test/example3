Start : Start
Terminals : hello_world
Start   ->  hello_world:a b

