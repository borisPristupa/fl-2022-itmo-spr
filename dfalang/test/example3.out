Alphabet: a b   
States: Start hello_world   
Start: Start    
Terminals: hello_world
Start -> hello_world: a b
